pkg/jobrunaggregator/jobrunaggregatorlib/jobrun_locator_release_jobs.go

```diff
@@ -36,11 +43,17 @@ func NewPayloadAnalysisJobLocatorForReleaseController(
 }
 
 type releaseControllerProwJobMatcher struct {
-	payloadTag string
+	payloadTag   string
+	isAggregated bool
```

Comment:
you're plumbing this through, but as we discussed on slack with brad, the distinction doesn't matter, so this is adding complexity without benefit.

Reply:
As of now, aggregator is only coded to work with 10 jobs. So we need to differentiate aggregated and non-aggregated jobs. Otherwise, aggregator will panic when processing old data.

Reply:
Done